### PR TITLE
Issue #13321: Kill mutation for JavadocUtil

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13321: Kill mutation for JavadocUtil

------

# Mutation 
https://github.com/checkstyle/checkstyle/blob/bb64c20cbcf6e543b297b6e809f379bd8746b5e7/config/pitest-suppressions/pitest-utils-suppressions.xml#L291-L299

------

# Explaination
Their is no usage of this method in whole codebase

### CODE REVIEW

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
I am good to merge, this is internal utility method.

File: config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml [hunk 1]
```diff
@@ -6205,7 +6201,6 @@
     </details>
   </checkerFrameworkError>
 
-  <!-- The following 2 fields are initialized during parsing step -->
```
Comment (romani) [inline config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml:6208]: Is there way to keep such comments?

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml (hunk 1 above)
Comment (Kevin222004) [reply config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml]: Comments restored. Actually I have not noticed that it has been removed

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
I merging, as we just remove whole method.

**Pull request merged**